modules/ROOT/pages/creating-an-odatav4-api-with-apikit.adoc

```diff
@@ -319,7 +203,7 @@ curl --location --request PATCH 'http://localhost:8081/api/Orders(OrderID=10251,
 }'
 --
 
-OData v4 discourages the usage of PUT to update your entities because it replaces the entire entity with a new one risking you a possible data loss.
+OData V4 discourages the usage of PUT to update your entities because it replaces the entire entity with the new one, which can lead to data loss.
```

Comment:
```suggestion
OData V4 discourages the usage of PUT to update your entities because PUT replaces the entire entity with the new one, which can lead to data loss.
```

Reply:
OK! Thanks!